wpilibc/src/main/native/cpp/RobotBase.cpp

```diff
@@ -65,7 +83,30 @@ class WPILibCameraServerShared : public frc::CameraServerShared {
 }  // namespace
 
 static void SetupCameraServerShared() {
-  SetCameraServerShared(std::make_unique<WPILibCameraServerShared>());
+#ifdef __FRC_ROBORIO__
```

Comment:
What about CameraServer in simulation?

Reply:
All this does is provide the Hal usage reporting, and 1 other little flag. So in sim it doesn't really matter. Everything will still work.